### PR TITLE
feat: Add .tug command

### PR DESCRIPTION
**Pull request by xZreai**
Adds .tug command for people having issues stuck with the invisible pushback tug.

image

Discord: Cleon#5802

### CODE REVIEW

File: src/commands/support/tug.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+
+export const tug: CommandDefinition = {
+    name: ['tug', 'stuck'],
+    description: 'Help to screenshot for support',
```
Comment (oim1) [inline src/commands/support/tug.ts:8]:
```suggestion
    description: 'Provides a solution to an issue where the tug fails to disconnect.',
```

**Review by oim1 — CHANGES_REQUESTED** [review]

File: src/commands/index.ts [hunk 2]
```diff
@@ -150,10 +151,11 @@ const commands: CommandDefinition[] = [
     directx12,
     cfms,
     fdr,
-    discontinuity,
+    discontinuity,
```
Comment (benw202) [inline src/commands/index.ts:154]:
```suggestion
    discontinuity,
```

File: src/commands/support/tug.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+
+export const tug: CommandDefinition = {
+    name: ['tug', 'stuck'],
+    description: 'Provides a solution to an issue where the tug fails to disconnect.',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire Support | How to fix aircraft being stuck',
```
Comment (benw202) [inline src/commands/support/tug.ts:11]:
```suggestion
        title: 'FlyByWire Support | My Aircraft is Stuck!',
```

File: src/commands/support/tug.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        description: makeLines([
+            'Make sure you do not have the "NW STRG DISC" message on the upper ECAM display. '
+           + 'If you do, please press "Shift" + "P" on your keyboard to disconnect the invisible pushback tug.',
```
Comment (benw202) [inline src/commands/support/tug.ts:14]:
```suggestion
           'If you do, please press "Shift" + "P" on your keyboard to disconnect the invisible pushback tug.',
           '',
```

File: src/commands/support/tug.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+
+export const tug: CommandDefinition = {
+    name: ['tug', 'stuck'],
+    description: 'Provides a solution to an issue where the tug fails to disconnect.',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire Support | How to fix aircraft being stuck',
+        description: makeLines([
+            'Make sure you do not have the "NW STRG DISC" message on the upper ECAM display. '
```
Comment (benw202) [inline src/commands/support/tug.ts:13]:
```suggestion
            'Make sure you do not have the "NW STRG DISC" message on the upper ECAM display. ',
```

File: src/commands/index.ts (hunk 2 above)
Comment (Benjozork) [inline src/commands/index.ts:154]: **style:** please revert

File: src/commands/support/tug.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
```
Comment (Benjozork) [inline src/commands/support/tug.ts:4]: **style:** extra newline

File: src/commands/support/tug.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    description: 'Provides a solution to an issue where the tug fails to disconnect.',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire Support | How to fix aircraft being stuck',
+        description: makeLines([
+            'Make sure you do not have the "NW STRG DISC" message on the upper ECAM display. '
+           + 'If you do, please press "Shift" + "P" on your keyboard to disconnect the invisible pushback tug.',
+             'Please read our guide [here](https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#nose-wheel-steering-locked-nw-strg-disc) for more information. ',
+        ]),
+
```
Comment (Benjozork) [inline src/commands/support/tug.ts:17]: **style:** extra newline

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: src/commands/support/tug.ts (hunk 7 above)
Comment (benw202) [inline src/commands/support/tug.ts:17]:
```suggestion
```

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/commands/support/tug.ts [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+import { CommandCategory } from '../../constants';
+
+export const tug: CommandDefinition = {
+    name: ['tug', 'stuck'],
+    description: 'Provides a solution to an issue where the tug fails to disconnect.',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire Support | My Aircraft is Stuck!',
+        description: makeLines([
+                'Make sure you do not have the "NW STRG DISC" message on the upper ECAM display. ',
```
Comment (benw202) [inline src/commands/support/tug.ts:12]:
```suggestion
            'Make sure you do not have the "NW STRG DISC" message on the upper ECAM display. ',
```

File: src/commands/support/tug.ts [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+                'If you do, please press "Shift" + "P" on your keyboard to disconnect the invisible pushback tug.',
```
Comment (benw202) [inline src/commands/support/tug.ts:13]:
```suggestion
            'If you do, please press "Shift" + "P" on your keyboard to disconnect the invisible pushback tug.',
```

File: src/commands/support/tug.ts [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+                'Please read our guide [here](https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#nose-wheel-steering-locked-nw-strg-disc) for more information. ',
```
Comment (benw202) [inline src/commands/support/tug.ts:14]:
```suggestion
            'Please read our guide [here](https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/#nose-wheel-steering-locked-nw-strg-disc) for more information. ',
```

**Review by benw202 — APPROVED** [review]

**Review by Benjozork — APPROVED** [review]

**Pull request merged**